specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -1228,6 +1228,283 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the detail of the linked workspace.",
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspace"
+            }
+          },
+          "404": {
+            "description": "Specific linked workspace not found.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "LinkedWorkspaces"
+        ],
+        "operationId": "LinkedWorkspaces_Update",
+        "description": "Update the detail of a linked workspace.",
+        "x-ms-examples": {
+          "UpdateLinkedWorkspace": {
+            "$ref": "./examples/updateLinkedWorkspace.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkName"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The object for updating an existing linked workspace",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspaceUpdateDto"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the detail of the linked workspace.",
+            "schema": {
+              "$ref": "#/definitions/LinkedWorkspace"
+            }
+          },
+          "404": {
+            "description": "Specific linked workspace not found.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/Microsoft.MachineLearningServices/workspaces/{workspaceName}/delinkWorkspace/{linkName}": {
```

Comment:
>delinkWorkspace [](start = 133, length = 15)

Are you introducing a separate nested resource type for de-linked workspaces? Did you mean to create a delink POST action on a linked workspace instead?

Reply:
User can link multiple LinkedWorkspace to Workspace, here is to delink one of them, i.e. delete a LinkedWorkspace. If DELETE is not reserved for resource, I will use DELETE .../linkedWorkspace/{linkName} here. And POST .../linkedWorkspace/{linkName} is used by the update, so I use .../delinkWorkspace/{linkName} here for delete.